README.md

```diff
@@ -0,0 +1,2 @@
+# .github
+[OpenMBEE](https://github.com/Open-MBEE) organization health files
```

Comment:
I think this can be removed.

Reply:
I think we should keep it, just to have a README.md for the repo itself.